src/formgradernext/formgradernext/handlers.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+
+import requests
+from jinja2 import BaseLoader
+from jinja2 import Environment
+from nbgrader.server_extensions.formgrader.base import BaseHandler
+from nbgrader.server_extensions.formgrader.base import check_notebook_dir
+from nbgrader.server_extensions.formgrader.base import check_xsrf
+from notebook.notebookapp import NotebookApp
+from notebook.utils import url_path_join as ujoin
+from tornado import web
+
+from .middleware import coop_coep_headers
+
+DEFAULT_LMS_VERSION = os.environ.get("DEFAULT_LMS_VERSION") or "0.3.0"
+
+
+def get_template(version: str) -> str:
+    """Gets the template based on the version of the LMS package."""
+    template_response = requests.get(
+        f"https://content.illumidesk.com/lms/{version}/index.html"
+    )
+    template_html = template_response.text.replace(
+        "</head>",
+        """
+            <script>
+                var base_url = "{{ base_url }}";
+                var url_prefix = "{{ url_prefix }}";
+                var base_url_suffix = "{{ base_url_suffix }}";
+            </script>
+        </head>
+        """,
+    )
+    # Hack(@gzuidhof): We need this until the index file in the bundle itself contains crossorigin (or crossorigin="anonymous") tags
+    # we need to specify crossorigin assets specifically due to the COOP and COEP headers.
+    return template_html.replace("<script ", "<script crossorigin ").replace(
+        "<link ", "<link crossorigin "
+    )
+
+
+class LMSHandler(BaseHandler):
+    @web.authenticated
+    @check_xsrf
+    @check_notebook_dir
+    @coop_coep_headers
+    def get(self):
+        html = (
+            Environment(loader=BaseLoader)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

HTML templating engines have an `autoescape` mechanism that protects web applications from the most common cross-site-scripting (XSS) vulnerabilities. To increase the security of your code, set the `autoescape` attribute to `True`.